### ISSUE TITLE
[BUG] Buy and sell orders getting out of sync

### ISSUE THREAD

**Issue by Clint-G13** [CoinAlpha/hummingbot#1446]
**Describe the bug**
When running pure market making strategy with multiple orders, buy and sell orders are out of sync after few times that order gets refreshed with no orders getting filled. The delay period starts from 1 sec to 10 seconds.

**Steps To Reproduce**
Here are the steps to reproduce the issue (see attachments in the section below):
1. Create a pure market making strategy and set mode to Multiple
2. Start bot and wait for orders to get refreshed.
3. After 2-4 refreshed, orders will start to de-sync. Initially, there will be 1 sec delay and the longer you wait, the delay time will increase.

Note: The user reported that same behavior occurred with KuCoin.
...

**Screenshots**
![unknown_(1)](https://user-images.githubusercontent.com/57189990/75050926-eac2fa00-5507-11ea-9569-20a4c60c4af9.png)
![Binance_multiple_order](https://user-images.githubusercontent.com/57189990/75050931-ec8cbd80-5507-11ea-9721-f3f63faa882c.png)
![Binance_out_of_sync](https://user-images.githubusercontent.com/57189990/75050938-ee568100-5507-11ea-8d11-75542785a20c.png)

**Release version**
Environment: Windows Binary, Docker (Linux)
Master and Dev branch
```
Last test
branch: development
commit ab4ad160db96fa814541d152718215d8160e95ed
Date:   Tue Feb 18 10:11:23 2020 -0800
```
**Attachments**
[logs_conf_pure_market_making_strategy_31.log](https://github.com/CoinAlpha/hummingbot/files/4236941/logs_conf_pure_market_making_strategy_31.log)

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:** VoRteX
**Optional: your ETH address:** (If you want to participate in the bug bounty)

**Comment by MishaelTereso** [comment]
Tested this BUG using the same configuration (except for the token pair since I don't have RLC & ZRX) via WSL with the updated release image on dev branch 0.24.0 and did not reproduce the behavior.

![image](https://user-images.githubusercontent.com/56957119/75137519-c2fbae00-5722-11ea-99ff-b95cfc9c4f9d.png)

![image](https://user-images.githubusercontent.com/56957119/75137602-02c29580-5723-11ea-957b-120b4e0c5523.png)

![image](https://user-images.githubusercontent.com/56957119/75137763-6c42a400-5723-11ea-916f-565b455ca705.png)

![image](https://user-images.githubusercontent.com/56957119/75138135-73b67d00-5724-11ea-988d-7638891ace9e.png)

**Comment by rennel-tabing** [comment]
I am able to reproduce this behavior on 
version: dev-0.26.0

My bot had the behavior of a 4 seconds difference between the buy and the sell order after running a strategy for 30 minutes. The first few minutes of running the strategy gave me a 1 second delay between the buy and the sell.

![image](https://user-images.githubusercontent.com/53868268/78919295-b7104500-7a5f-11ea-952d-71bb09a6f878.png)

On the recorded video,
You will notice that the bot is also placing the new order with a 4 second delay on the other side of the order book. 
https://vimeo.com/user112309579/review/405906882/e53b118cbe

**Comment by dennisocana** [comment]
This also happens on single order level.

**Comment by dennisocana** [comment]
This also happens in Liquid and probably to other connectors too.

![image](https://user-images.githubusercontent.com/50150287/80558963-b4758f80-8a0e-11ea-8ff3-c85bef5d18ff.png)

**Comment by mifeng** [comment]
Fixed in #1673
